### DISCUSSION TITLE
Anchors Questions

### DISCUSSION THREAD

**Discussion by vincent-reau** [lululxvi/deepxde#1625] [Q&A]
I use [Discussion #1556](https://github.com/lululxvi/deepxde/issues/1556) to have a code example.

I'm trying to understand the difference between anchors  we create with PointSetBC like below:
`observe_u = dde.icbc.PointSetBC(ob_xy, ob_u, component = 0)` 
`observe_v = dde.icbc.PointSetBC(ob_xy, ob_v, component = 1)`
`observe_p = dde.icbc.PointSetBC(ob_xy, ob_p, component = 2)`

and the anchors in dde.data.PDE()

`data = dde.data.PDE(`
`    geom,`
`    Navier_Stokes_Equation,`
`    [observe_u, observe_v, observe_p],`
`    # num_boundary = 200,`
`    # num_domain = 2500,`
`    anchors = ob_xy,`
`)`

Could someone try to explain both in simple terms but also in mathematical terms ?

Thanks

**Comment by ruchitkini** [comment]
I think in Discussion #1556, the known data has been used. So for training the known data points which are coordinates x and y are in the set ob_xy and the output value of velocities and pressure on those points are in set ob_u, ob_v, and ob_p respectively. So in the second code snippet, the anchors given are the coordinates on which model is to be trained, and in the first code snippet, the pointSetBC is used to calculate the loss using known output values of u, v, and p, on the same points given as anchors, which is then included in dde.data.pde as boundary conditions.
